allay/parser.py

```diff
@@ -18,15 +18,12 @@ def parse(self, contents: str, indent: int = None) -> str:
         Returns:
             str: The text-component
         """
-        try:
-            if indent:
-                return json.dumps(
-                    self.internal_parse(TokenStream(contents)), indent=indent
-                )
-            else:
-                return json.dumps(self.internal_parse(TokenStream(contents)))
-        except InvalidSyntax as error:
-            raise SyntaxError(error.format("src"))
+        if indent:
+            return json.dumps(
+                self.internal_parse(TokenStream(contents)), indent=indent
+            )
+        
+        return json.dumps(self.internal_parse(TokenStream(contents)))
```

Comment:
You took out the try except block here, so any syntax errors won't be reported.

Reply:
they will be reported as InvalidSyntax errors by tokenstream, there's no need to catch it and then put out a new exception. i can add it back tho